backend/src/controllers/question.js

```diff
@@ -1,22 +1,71 @@
 const questionModel = require('../models/question')()
 const commentModel = require('../models/comment')()
+const likesModel = require('../models/likes')()
 const errorHandler = require('./error')
 const { responseHandler } = require('../utils/helpers')
 
 /* GET question data. */
 exports.getQuestion = function ({ params }, res) {
-    responseHandler(questionModel.getQuestion(params.id), res)
+    const getQuestion = Promise.all([
+        questionModel.getQuestion(params.id),
+        likesModel.getLikes({ type: 'question', id: params.id })
+    ])
+        .then(([question, likes]) => { return { ...question, ...likes } })
```

Comment:
This will spread the object into a flat structure. What does the likes object look like?

Reply:
Just a normal question object with the addition of likes